src/parser/warlock/demonology/modules/pets/helpers.js

```diff
@@ -2,5 +2,7 @@ import PETS from './PETS';
 
 export const isPermanentPet = guid => guid.toString().length > 6;
 export const isWildImp = guid => guid === PETS.WILD_IMP_HOG.guid || guid === PETS.WILD_IMP_INNER_DEMONS.guid;
+export const isDreadstalker = guid => guid === PETS.DREADSTALKER.guid;
```

Comment:
These two are no longer necessary as they're not used anymore

Reply:
Fixed.